src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/AggregationIT.java

```diff
@@ -448,6 +454,585 @@ public void countGroupByDateTestWithAlias() throws IOException {
 //        // TODO: fix the query or fix the code for the query to work
 //    }
 
+    @Test
+    public void topHitTest() throws IOException {
+
+        String query = String.format("select topHits('size'=3,age='desc') from %s group by gender", TEST_INDEX_ACCOUNT);
+        JSONObject result = executeQuery(query);
+        JSONObject gender = getAggregation(result, "gender");
+        Assert.assertThat(gender.getJSONArray("buckets").length(), equalTo(2));
+
+        final int maleBucketId;
+        final int femaleBucketId;
+
+        if ((gender.query("/buckets/0/key")).equals("m")) {
+            maleBucketId = 0;
+            femaleBucketId = 1;
+        } else {
+            maleBucketId = 1;
+            femaleBucketId = 0;
+        }
```

Comment:
```suggestion
        int maleBucketId = 1;
        int femaleBucketId = 0;

         if ((gender.query("/buckets/0/key")).equals("m")) {
            maleBucketId = 0;
            femaleBucketId = 1;
        }
```
Also may put maleBucketId and femaleBucketId to @Before, as looks like these two variables are used in a few cases.

Reply:
Don't want to put it in @before since it's not common to most of the tests. WIll change to use a ternary operator like:  
```java
final boolean isMaleFirst = gender.query("/buckets/0/key").equals("m");
final int maleBucketId = isMaleFirst ? 0 : 1;
final int femaleBuketId = isMaleFirst ? 1 : 0;
```